bin/process_partis.py

```diff
@@ -111,7 +111,17 @@ def merge_upstream_seqmeta(partis_seqmeta, upstream_seqmeta):
             dup_upstream_row = get_upstream_row(upstream_seqmeta, seqid)
             # pre-partis filtering multiplicity
             dup_multiplicity = int(dup_upstream_row.get('multiplicity', 1))
-            timepoints_dict[dup_upstream_row.get('timepoint')] += dup_multiplicity
+            if not dup_upstream_row.get('timepoint'):
+                should_not_have_timepoint = seqid in seqs_without_timepoints # stored as a variable for readability on the next line
+                if ignore_timepoint_info or should_not_have_timepoint:
+                    # if we explicitly run with --ignore-timepoint-info, or the sequence is not expected to have timepoint
+                    # info, like a naive sequence (or seed?), then we assign a dummy timepoint so multiplicity is calculated even in the absence of timepoint data.
+                    dup_upstream_row['timepoint'] = 'no-timepoint'
```

Comment:
@eharkins Do we want `dup_upstream_row` or `upstream_row` here? Look's like we're using the latter below in defining `result_row`.

Reply:
I used `dup_upstream_row` because I thought we still want to be using that dict at this point since here https://github.com/matsengrp/cft/pull/269/files/630bba67ccf7f417710aa9b1023e9f588d0c1087#diff-b093a3f55dad4e7a5edae3d2a2bf4396R124 is where we track the multiplicity. So using `dup_upstream_row` allows that line to apply to rows with and without timepoints. Does that make sense? You could still be right

Reply:
I think you're right regarding the `timepoints` attribute in `results_row` (line 126/136 in the diff below). But `timepoint` (line 120/130) is defined in terms of `upstream_row`.

Reply:
Yep, you're right. Thanks! Maybe I'll move the dummy timepoint default into `get_upstream_row()` for DRYness